libs/Microsoft.MixedReality.WebRTC.Native/src/interop/interop_api.cpp

```diff
@@ -5,6 +5,7 @@
 // line, to prevent clang-format from reordering it with other headers.
 #include "pch.h"
 
+#include "api/stats/rtcstats_objects.h"
```

Comment:
Can you please separate core headers from mr-webrtc ones with a blank line?

Reply:
Done